core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -921,8 +956,22 @@ var ExplorationEditorMainTab = function() {
             ') is expected to match # of placeholders(' +
             (placeholders.length) + ')');
           }
-          ruleDescription = ruleDescription.replace(
-            placeholderElement, providedText[index].toString());
+          // In the case of MultipleChoiceInput, the rule description is
```

Comment:
I'm surprised that this consideration is only coming up now. Why didn't it cause an issue before?

Reply:
this function is only used by `expectRuleToBe`, But previously we didn't use this function for testing the case of 'MultiChoiceInput'. That's why it didn't cause an issue before.

Reply:
Ah ok. Thanks for explaining!